prime-router/src/main/kotlin/Receiver.kt

```diff
@@ -232,8 +232,18 @@ open class Receiver(
      * Validate the object and return null or an error message
      */
     fun consistencyErrorMessage(metadata: Metadata): String? {
-        // TODO: Temporary workaround for full-ELR as we do not have a way to load schemas yet
-        if (topic == Topic.FULL_ELR) return null
+        if (topic == Topic.FULL_ELR) {
+            if (conditionFilter.isNotEmpty()) {
+                conditionFilter.forEach {
```

Comment:
This check is not needed. The condition filter can be any FHIR Path expression and you are limiting that here.